src/rebound.h

```diff
@@ -577,6 +577,8 @@ enum REB_BINARY_FIELD_TYPE {
     REB_BINARY_FIELD_TYPE_IAS15_MINDT = 70,
     REB_BINARY_FIELD_TYPE_IAS15_EPSILONGLOBAL = 71,
     REB_BINARY_FIELD_TYPE_IAS15_ITERATIONSMAX = 72,
+    REB_BINARY_FIELD_TYPE_IAS15_DTMODE = 73,
```

Comment:
I'm not quite sure if allocating 73 and 74 here for the new variables would incur any problem (i.e. backward compatibility with loading previous versions of REBOUND simulation archives)...

Reply:
You're right, this could be a problem (I doubt many would run into it). Choose a number higher than the highest one used.

Reply:
It's now updated to 401 and 402.